glommio/src/lib.rs

```diff
@@ -29,12 +34,14 @@
 //!     }).unwrap();
 //! ```
 //!
-//! Although this is not forced upon the user, by creating N executors and binding
-//! each to a specific CPU one can use this crate to implement a thread-per-core
-//! system where context switches essentially never happen, achieving maximum efficiency.
+//! ### Pinned threads
```

Comment:
I added in some section headers and changed some wording through here. Unfortunately, I wasn't really sure on the line-lengths being used, and did some line-wrapping at 100 characters; this means that its hard to tell from the diff what did and did not change.

Reply:
My philosophy on style is that if a machine does not complain a human shouldn't either.
So whatever passes `cargo clippy` and `cargo fmt` is acceptable to me.

Yes, I fully acknowledge that a consistent code base is desirable (and `cargo clippy` and `cargo fmt` do a great job at that) but as with most things in life it comes with hidden costs: in this case, all the time that is spent with people obsessing and fighting over personal preferences.

tl;dr: do as you please!

Reply:
I liked the section headers, it is a nice touch.

Thanks a lot for this, Wendell.